.github/workflows/pr-check_markdownlint.yml

```
@@ -0,0 +1,37 @@
+name: Markdownlint (PR files)
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - "**/*.md"
+
+jobs:
+  docs:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Get changed files
+        id: changed-files
+        uses: tj-actions/changed-files@v29
+        with:
+          files: |
+            **/*.md
+
+      - name: Setup Node.js environment
+        uses: actions/setup-node@v3
+        with:
+          node-version: "16"
+          cache: yarn
+
+      - name: Install all yarn packages
+        run: |
+          yarn --frozen-lockfile
+
+      - name: Lint markdown files
+        run: |
+          echo "::add-matcher::.github/workflows/markdownlint-problem-matcher.json"
+          yarn markdownlint-cli2 ${{ steps.changed-files.outputs.all_changed_files }}
```

```suggestion
          echo "::add-matcher::.github/workflows/markdownlint-problem-matcher.json"
          files_to_lint="${{ steps.changed-files.outputs.all_changed_files }}"
          yarn markdownlint-cli2 ${files_to_lint}
```
There are 66 files in the repo with `(` in the path.